scripts/ci-build-azure-ccm.sh

```diff
@@ -36,6 +36,8 @@ export CCM_IMAGE_NAME=azure-cloud-controller-manager
 export CNM_IMAGE_NAME=azure-cloud-node-manager
 declare -a IMAGES=("${CCM_IMAGE_NAME}" "${CNM_IMAGE_NAME}")
 
+WINDOWS_VERSION=1809
```

Comment:
we have an environment called `WINDOWS_SERVER_VERSION` which is being set for many of the templates like https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/f90058dfcfac69dec29318367810cf82bdcee921/templates/test/ci/prow-ci-version/patches/machine-deployment-ci-version-windows.yaml#L14

what do you think of using that value to calculate the `1809` or `ltcs2022`?  I just worry about having to many similar env variables that almost do the same thing.  @marosset thoughts?

Reply:
I agree

Reply:
Thanks! I will take a look.